pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scanapi"
-version = "1.0.5"
+version = "1.0.7"
```

```suggestion
version = "1.0.5"
```
Since we will not bumpo the version now, we can leave this at is.